Hammerspoon/MJConsoleWindowController.m

```diff
@@ -67,10 +79,23 @@ - (void) setup {
 }
 
 - (void) reflectDefaults {
+    
+    //
+    // Dark Mode:
+    //        
+    if (ConsoleDarkModeEnabled()) {
+        self.window.appearance = [NSAppearance appearanceNamed: NSAppearanceNameVibrantDark] ;
+        self.window.titlebarAppearsTransparent = YES ;
+    } else {
+        self.window.appearance = [NSAppearance appearanceNamed: NSAppearanceNameVibrantLight] ;
+        self.window.titlebarAppearsTransparent = NO ;
```

Comment:
self.outputView.enclosingScrollView.drawsBackground = YES ;

Adding these two lines seems to make the scrollbar more reasonable when in dark mode.  I'd leave changing the actual color of the background to `hs.console.outputBackgroundColor` and `hs.console.consoleCommandColor`, etc. since the logic gets confusing trying to determine if they should be changed or not based upon the users possibly already chosen theming.

Reply:
Though maybe mention them in the docs for `hs.console.darkMode` documentation.